api/scpca_portal/models/job.py

```diff
@@ -104,36 +111,125 @@ def get_sample_job(
             },
         )
 
+    @classmethod
+    def submit_created(cls, list_of_jobs: List[Self]) -> bool:
+        """
+        Submit the given unsaved created jobs via batch.submit_job.
+        Update each job instance's batch_job_id and state, and
+        save it to the db on success.
+        """
+        submitted_jobs = []
+        unsaved_jobs_found = False  # Track if at least one unsaved job in list_of_jobs
+        failed_count = 0
+
+        for job in list_of_jobs:
+            if job.state != JobStates.CREATED.name:
+                continue
+
+            unsaved_jobs_found = True
+
+            if aws_job_id := batch.submit_job(job):
+                job.batch_job_id = aws_job_id
+                job.state = JobStates.SUBMITTED.name
+                job.submitted_at = make_aware(datetime.now())
+                submitted_jobs.append(job)
+            else:
+                failed_count += 1
+
+        if not unsaved_jobs_found:
+            logger.info("No submission required as all jobs were previously submitted.")
+            return True
+
+        if submitted_jobs:
+            Job.objects.bulk_create(submitted_jobs)
+
+            # TODO: How to handle logging?
+            total_submitted_count = len(submitted_jobs)
+            logger.info(
+                "Job submission complete. "
+                f"{total_submitted_count} job{pluralize(total_submitted_count)} were submitted.",
+            )
+            return True
+
+        # TODO: How to handle logging?
+        if failed_count > 0:
+            logger.info(f"Failed to submit {failed_count} job{pluralize(failed_count)}.")
+
+        return False
+
+    @classmethod
+    def terminate_submitted(cls, retry_on_termination: bool = False) -> bool:
```

Comment:
Update this to match error handling above. Also, remove logging as batch.terminate_job should handle that.